Utils/Dataflow/010_glancePapers/papersFromGLANCE.sh

```diff
@@ -93,34 +122,35 @@ if ! krenew -H 60 ; then
   if [ -n "$RETRY" ]; then
     kinit "$USR@CERN.CH"
   else
-   echo "No Kerberos ticket for CERN.CH realm found. Exiting." >&2
+   echo "(ERROR) No Kerberos ticket for CERN.CH realm found. Exiting." >&2
    exit 3 
   fi
 fi
-klist | grep 'CERN.CH' >/dev/null || { echo "No Kerberos ticket for CERN.CH realm found. Exiting." >&2; exit 3; }
+klist | grep 'CERN.CH' >/dev/null || { echo "(ERROR) No Kerberos ticket for CERN.CH realm found. Exiting." >&2; exit 3; }
 
 # Lxplus address
 lxplus=lxplus.cern.ch
 
 option="-q -o StrictHostKeyChecking=no -o GSSAPIAuthentication=yes -o GSSAPIDelegateCredentials=yes -o GSSAPITrustDNS=yes"
 ssh $option -K $USR@$lxplus "( $cmd1; $cmd2; ) 2>&1 >/dev/null" 2>&1 > /dev/null
-[ $? ] || { echo "Can not execute remote commands. Exiting." >&2; exit 4; }
+[ $? ] || { echo "(ERROR) Can not execute remote commands. Exiting." >&2; exit 4; }
 
 tmp=/tmp/list_of_papers.json
 scp $option $USR@$lxplus:$json $tmp >&2
 
-[ $? ] || { echo "Can not copy file from remote. Exiting." >&2; exit 4; }
+[ $? ] || { echo "(ERROR) Can not copy file from remote. Exiting." >&2; exit 4; }
 
-echo -ne "\n" >> $tmp
+echo -ne "$EOMessage" >> $tmp
 
 if [ -n "$FILE" ] ; then
   cat $tmp > $FILE
 else
   cat $tmp
 fi
 
+echo -ne "$EOProcess"
```

Comment:
Remind me about this, please.
Have we decided to send EOP to STDOUT independedntly of the destination of output data (file, pipe, STDOUT, whatever), separating so-called "command channel" from "data channel"?

Reply:
@mgolosova 

While we have only 1 message (so-called JSON-array) EOM and EOP should be just sent after that message. You said that I can write both of them into $tmp-file, (or) but it would be better to leave only EOM in $tmp and to output EOP explicitly (just to be prepared for a case when we'll divide somehow "data channel" and "command channel" . :)

Reply:
Right, now I remember it. Thank you!